lib/cisco_node_utils/version.rb

```diff
@@ -14,5 +14,5 @@
 
 # Container module for version number only.
 module CiscoNodeUtils
-  VERSION = '1.0.2-dev'
+  VERSION = '1.0.2'
```

Comment:
Hey Glenn,

Yes, sorry, this is unintended. However, we need to change this otherwise gem building fails. I will remove this though for the sake of this PR.

Thanks

JT